python/test/t_IndependentMetropolisHastings_std.py

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+import openturns as ot
+import openturns.testing as ott
+
+ot.TESTPREAMBLE()
+
+# %%
+# Draw the unnormalized probability density
+# -----------------------------------------
+
+import openturns as ot
+from numpy import pi
+
```

Comment:
```suggestion
```
OpenTURNS has already been imported. It is strange that numpy cannot be imported, but perhaps "math" can be (cf. my suggestion above).